src/plugins/data/common/search/es_search/types.ts

```diff
@@ -31,5 +31,13 @@ export interface IEsSearchRequest extends IKibanaSearchRequest {
 }
 
 export interface IEsSearchResponse extends IKibanaSearchResponse {
+  /**
+   * Indicates whether async search is still in flight
+   */
+  isRunning?: boolean;
```

Comment:
It probably makes sense to either move these directly to `IKibanaSearchResponse`, or move `id` inside here. Maybe we can have a conversation about what should be in `IKibanaSearchResponse` (or if it's necessary).